srcpkgs/spotify-adblock/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'spotify-adblock'
+pkgname=spotify-adblock
+version=1.0.1
+revision=1
+build_style=gnu-makefile
```

Comment:
the makefile literally just runs cargo build, so I see no reason to use the makefile style over the cargo style